### ISSUE TITLE
Checks for selected year and subject

### ISSUE THREAD

**Issue by devv77** [bprof-projm/foerum#116]

**Comment by conoren** [comment]
Additional info: topics do not have yearID in them, if you want to filter them by year, you must extract the year from the creationDate.

**Comment by conoren** [comment]
But as I see, a server-side solution for this is being implemented.

**Comment by conoren** [comment]
https://codesandbox.io/s/r5x4i?file=/src/App.js:679-692
this can be of help

**Comment by devv77** [comment]
@IIB3nceII can you check this on BE side? Should we implement the year id? But as far as I know, this issue focusing on choosing a year and filter it to subjects and displaying the newest topics on the feed. I dont know if we need the year id, its just has to be in order by the created date. Or am i just misunderstanding something? @HegeMte any thoughts on this one? 
Thanks @conoren

**Comment by devv77** [comment]
#105 this has something to do with this issue. Just linking them together

**Comment by HegeMte** [comment]
yes, the functionality of the function is well described by David, (For clarification: it lists the topics related to the given subject in chronological order. The topic and the subject itself are linked, as you can see in the picture.) @IIB3nceII  why should we add a yearID? @devv77  @conoren 
![image](https://user-images.githubusercontent.com/74251371/163451486-b2ba00ce-d092-4c4d-95e5-e77934daba53.png)

**Comment by IIB3nceII** [comment]
Yea they are linked, @conoren is there any other question?

**Comment by HegeMte** [comment]
just a quick question: why are we speaking English?

**Comment by IIB3nceII** [comment]
Mert mindenki keszul kulfoldre🙈

**Comment by rolandbogdan** [comment]
Möchtest du auf Deutsch zu sprechen?  @HegeMte

**Comment by devv77** [comment]
> Yea they are linked, we can modify FE models if we must. @conoren is there any other question?

És egy két join-os query-vel simán visszaaadjuk a yearID-t is ha nagyon szükséges. Biztos @rolandbogdan  élvezné ha itt is írhatna egy kis querry-t. Amúgy akkor minden érthető, és lehet ezzel tovább haladni valamilyen módon. @IIB3nceII kérlek ha kéne segítség majd @conoren -nak, akkor legyél rá készen. Köszi
